### DISCUSSION TITLE
`kratos-client-go` hard implement?

### DISCUSSION THREAD

**Discussion by und3fined** [ory/kratos#1214] [Q&A] [answered]
Hi team,

My first question is `reorder fields` (https://github.com/ory/kratos/discussions/1196)
Currently, I get a problem with implement `kratos -client-go`
In one simple thought, I created a client from your documentation and reused `ory/kratos-client-go/models`.
Every day I only have about 1 to 2 hours of doing this. I probably won't have enough time to finish it.

So, i have a question `Do you have a plan to write documentation for kratos-client-go?`
Maybe, I can help for team write first document.

_Sorry, my english not good_

**Comment by vinckr** [comment, marked as answer]
There was a helpful comment on your other problem:
https://github.com/ory/kratos/discussions/1196#discussioncomment-584303 
Check out this code to order your fields:
https://github.com/ellioseven/kratos-ss-ui-react/blob/master/src/components/KratosForm.tsx

As for your other question:

We would love to have more documentation on the sdks, they are generated using openapi-generator, but that does not cover documentation of course. When we have more capacity for it, we will add more documentation there as well, for the moment being we are pretty busy getting Kratos to 1.0.

If you would like to contribute documentation, I am happy to help you along with it! 
Dont worry if your english is not good, we will find a way to make it work. 
Feel free to write me directly if you already have something documented :).

**Reply by vinckr** [reply]
Hey @und3fined , just checking in 🙂 
I will mark this as answered for now, but feel free to come back to the discussion any time.
Thanks a ton :bee:

**Reply by und3fined** [reply]
@vinckr thanks for reply
Currently, i'm work more job.
But in near future, i will come back.

I have more question `Do you have a plan for new  kratos-client-go sdk?`

**Reply by vinckr** [reply]
What do you mean by new kratos-client-go sdk? 
The sdk gets regular updates as we release new versions of Ory Kratos. Since we last spoke it has been updated many times: https://github.com/ory/sdk/tree/master/clients/kratos/go
